samples/client.conf

```diff
@@ -65,10 +66,9 @@ publish_timeout:
 min_publish_interval:
   secs: 0
   nanos: 100000000
-session_timeout: 60000
 performance:
   ignore_clock_skew: false
   recreate_monitored_items_chunk: 1000
-session_name: Rust OPC UA Client
-channel_lifetime: 60000
 recreate_subscriptions: true
```

Comment:
not sure why that file was changed. I think it it rustfmt playing around. seems to be only ordering so I jsut leave it

Reply:
This file is generated when running the tests, so I assume some things have been changed/reordered in other PRs but this file was not added to those PRs. So this is fine and only indicated you've run the test 😉